### PR TITLE
More descriptive exception when the bundled-extension class does not exist 

### PR DESCRIPTION
**Pull request by lenvanessen**
A class thatt doesn't exist can't implement an interface, so this should be checked first.

### CODE REVIEW

**Comment by lenvanessen** [comment]
fixes #6836

File: src/Bootstrap.php [hunk 1]
```diff
@@ -158,6 +158,9 @@ public static function run($rootPath)
                 'extensions',
                 function ($extensions) use ($config) {
                     foreach ((array) $config['extensions'] as $extensionClass) {
+                        if(! class_exists($extensionClass) ) {
+                            throw new LogicException("$extensionClass is defined in bolt.yml, but class is misspelled or is not loaded by composer");
+                        }
```
Comment (GawainLynch) [inline src/Bootstrap.php]: This will throw exceptions for instantiated classed.

Bundles are expert level, from the beginning by design, this is out of scope

**Comment by GawainLynch** [comment]
Reopening, as now I have more time to reply (plus what I mentioned in Slack), lets get this over the line. My reply was too terse :laughing: 

I can do some magic for you to get this where it needs to be, or grab me again in Slack and lets go over it with more detail :+1:

**Comment by GawainLynch** [comment]
As mentioned in Slack, I've rebased the branch to pull in the tests from #6844 and tweaked the location of the check, plus added a test for it.

Catch you when you're back :+1:
